### PR TITLE
Handle node networkType, fixes #484 

### LINKED ISSUE #484: Prompt whether the user node network matches the account network type
Warn or prohibit when the account uses a node that does not match its network type

### PR DESCRIPTION
**Pull request by NemTechCN**
image
Warn when the account uses a node that does not match its network type

### CODE REVIEW

File: src/language/en-US.ts [hunk 1]
```diff
@@ -810,5 +810,7 @@ export const en_US = {
     please_input_mosaic_and_amount:'please input mosaic and amount',
     refresh_too_fast_warning:'The refresh rate is too high, please try again later.',
     display_mnemonic_QR_code:'Display mnemonic QR code',
-    fromwhere:'from'
+    fromWhere:'from',
+    Wallet_network_type_does_not_match_current_network_type:'Wallet network type does not match current node network type .'
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/language/en-US.ts]: fixed

File: src/core/services/listeners.ts [hunk 2]
```diff
@@ -165,4 +168,13 @@ export class ChainListeners {
...
 
+    async getNodeInfo() {
+        const store = this.app.$store
+        const node = store.state.account.node
+        const nodeHttp = new NodeHttp(node)
+        const nodeInfo = await nodeHttp.getNodeInfo().toPromise()
+        store.commit('SET_NODE_NETWORK_TYPE', nodeInfo.networkIdentifier)
+        console.log('ChainListeners node networkType ' + nodeInfo.networkIdentifier)
+    }
+
```
Comment (decentraliser) [inline src/core/services/listeners.ts]: Please move this one to service/network (it is not a Listener), and remove the console log

File: src/core/services/listeners.ts [hunk 3]
```diff
@@ -1,18 +1,21 @@
-import {Address, Listener} from "nem2-sdk"
+import {Address, Listener, NodeHttp, NodeInfo} from "nem2-sdk"
 import {filter} from 'rxjs/operators'
 import {formatAndSave} from '@/core/services/transactions'
-import {ChainStatus} from '@/core/model'
+import {AppState, ChainStatus} from '@/core/model'
+import {Store} from "vuex"
```
Comment (decentraliser) [inline src/core/services/listeners.ts]: Please mind unused imports

File: src/core/services/listeners.ts (hunk 2 above)
Comment (NemTechCN) [reply src/core/services/listeners.ts]: fixed

File: src/core/services/listeners.ts (hunk 3 above)
Comment (NemTechCN) [reply src/core/services/listeners.ts]: fixed

**Pull request merged**